packages/ui/src/working-groups/model/workingGroupName.ts

```diff
@@ -20,3 +20,17 @@ export const urlParamToWorkingGroupId = (name: string) => {
 }
 
 export const groupNameToURLParam = (name: string) => name.toLowerCase().replace(/ /g, '-')
+
+/**
+ *
+ * @param name
+ *  name has this shape workingGroupName-runtimeId
+ * @returns
+ *  workingGroupId
```

Comment:
This function is pretty simple plus its naming explains its purpose pretty well, so I don't see a reason behind this documentation. 

Plus, doesn't it return something else?
```suggestion
 *  workingGroupId-openingRuntimeId
```

Reply:
sorry, didn't even look at it. will add a commit to remove it.